libgraph/include/katana/PropertyGraph.h

```diff
@@ -962,11 +957,13 @@ class KATANA_EXPORT PropertyGraph {
     return *edge_entity_type_ids_;
   }
 
-  // This constructor is meant to be used is situations when you want to share
-  // property and type data (including RDG) with another PropertyGraph instance,
-  // while using a new topology. The topology is assumed to maintain the correct
-  // property index mapping from its nodes/edges to the original property table.
-  PropertyGraph(const PropertyGraph& parent, GraphTopology&& topo) noexcept
+  /// Call this constructor to populate projection data.
+  PropertyGraph(
```

Comment:
I'm saddened by addition of yet more fields to poor old dumpster of a `PropertyGraph`.

Reply:
At least the public interface didn't have to change much...